tests/conftest.py

```diff
@@ -1,9 +1,10 @@
 import pytest
+from cdisutilstest.code.conftest import indexd_server
+from indexclient.client import IndexClient
 
-from cdisutilstest.code.conftest import indexd_client, indexd_server
 
-@pytest.fixture(scope='function')
-def index_client(indexd_client):
+@pytest.fixture(scope='session')
```

Comment:
Why are you session scoping a client? This means all modifications that a test makes are going to be visible to other tests.
- Which down the line, means mocks could potentially be dependent on the order of test execution
- cdisutilstest also exposes and indexd_client (but I kinda think it should live in here?)

Reply:
Related PR from @philloooo : your not fully cleaning up indexd here:
https://github.com/uc-cdis/cdisutils-test/pull/12

Reply:
Right, usage of a shared client is the problem. If you are modifying what index client could return in one test a subsequent test *could* break. Additionally if one were to start mocking all requests and responses from IndexClient, the data returned would not be just for the test. Sharing the same instance of index_client is fine, but you must clean up all changes that the indexd_client made on a per test basis.

and cdisutilstest is ahead. you are cleaning 1/3 of the what you should be:
https://github.com/uc-cdis/cdisutils-test/pull/12/files#diff-2f85be2a3c761372f1109442ab7026f2R60